usr/share/rear/output/default/950_copy_result_files.sh

```diff
@@ -98,8 +98,22 @@ case "$scheme" in
     (fish|ftp|ftps|hftp|http|https|sftp)
         # FIXME: Verify if usage of $array[*] instead of "${array[@]}" is actually intended here
         # see https://github.com/rear/rear/issues/1068
+
         LogPrint "Copying result files '${RESULT_FILES[*]}' to $scheme location"
         Log "lftp -c open $OUTPUT_URL; $OUTPUT_LFTP_OPTIONS mput ${RESULT_FILES[*]}"
+
+        # OUTPUT_URL for lftp transfers will look something like:
+        # "<fish|ftp|ftps|hftp|http|https|sftp>://<host_name>/<destination_directory>"
+        # Using `cut' for printing 4th field with "/" delimiter (.. | cut -d/ -f4-),
+        # will strip leading "<protocol>://<hostname>/" from OUTPUT_URL
+        # and by prefixing "/" we will get absolute path to destination directory.
+        local dest_dir="/$(echo $OUTPUT_URL | cut -d/ -f4-)"
```

Comment:
Sorry - again I am wondering about `echo $OUTPUT_URL | cut -d/ -f4-`:
Doesn't our `url_path` function in lib/global-functions.sh do what you need?
See in particular the initial comment about all those `url_*` functions

Reply:
Fixed via b47bbca12dc25e208731f7e2480389cc478bfa0d